content/html/glossaries/html-tag-a.md

```diff
@@ -0,0 +1,65 @@
+---
+Title: "<a>"
+Subjects:
+  - "Web Development"
+  - "Web Design"
+Tags:
+  - "Anchor"
+  - "Link"
+  - "URL"
+Catalog Content:
+  - "https://www.codecademy.com/learn/learn-html"
+---
+
+## Definition 
+
+The HTML tag `<a>`, also known as the _anchor tag_, creates a link to another page or to a location in the current page. It can also be used to link to email addresses, phone numbers, and files.
+
+## Syntax
+
+```html
+<a href="url">Displayed Text</a>
+```
+
+`<a>` is a tag that commonly uses the following attributes to create a link from the current page to a valid URL:
+
+  - `href`, which represents the URL that the link points to
+  - `target`, which represents where to open and display the URL stored in the `href`
```

Comment:
This looks good for now. I wonder later if we should add an **Attributes** section either before or after **Examples** just to go over all the individual `target` keywords as well as other attributes.

Reply:
I think you could list subattributes right here?